### ISSUE TITLE
Piano doesn't sound well on Firefox Linux.

### ISSUE THREAD

**Issue by varishtsg** [Krazete/bookmarklets#2]
Playing the piano makes my computer sound as it were dying. Tried it on firefox on linux. The sound is horrible. Tried different waves as well. Please check your code. Tried it on an insecure site as well as localhost.

**Comment by Krazete** [comment]
Yeah, I've only really used it on my Mac. I've tested it on a few Windows computers and it sounds kinda off, so I figured it might sound even more horrible on some other computers.
Thanks for opening this issue; maybe someone more knowledgeable with AudioContext can fix it.
